client-js/app/elements/labrad-registry.html

```diff
@@ -97,6 +98,10 @@
     dir-item {
       white-space: nowrap;
     }
+    .mono {
+      font-family: 'roboto mono', monospace;
```

Comment:
I presume having the monospace fallback here means this will still use a monospace font (like courier) even if offline and can't connect to the net to download roboto mono. Have you tested that? I want to make sure that the app still works in an offline scenario, e.g. on lab computers that might not have access to the wider internet.